### DISCUSSION TITLE
Google Analytics 4 Support in 11ty Quire

### DISCUSSION THREAD

**Discussion by lizneely** [thegetty/quire#698] [Development]
Hello,

We recently released two publications (one title, two languages) on the new 11ty Quire. I would have loved to skip over UA Google Analytics and start with GA4, but it doesn't look like that's an option. UA is working on these publications, but we are speeding quickly to the July 1, 2023 phase-out date for that Google product. I considered going into the analytics.js component, but I'm a hack and I'd rather have it be in line with the core product.

Is GA 4 implementation on the roadmap?

Thanks,
Liz
Curator of Digital Experience
Georgia O'Keeffe Museum

**Comment by Erin-Cecele** [comment]
Thank you @lizneely. This is on our roadmap for v1. We will update the community as soon as possible.
